src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozerList = new ArrayList<>();
+
+    @Override
+    public List<Bulldozer> get() {
+        for (int i = 100; i < 105; i++) {
+            bulldozerList.add(new Bulldozer(i));
+        }
```

Comment:
Цього буде достатньо. Аналогічно з іншими
```suggestion
    @Override
    public List<Bulldozer> get() {
    	List<Bulldozer> bulldozerList = new ArrayList<>();
        for (int i = 0; i < 5; i++) {
            bulldozerList.add(new Bulldozer(i));
        }
```